catalogue/templates/catalogue/search.html

```diff
@@ -108,5 +149,19 @@
         });
       }
     });
+
+    document.getElementById("feedbackBtn").addEventListener("click", function() {
+      var modal = document.querySelector('.modal');
+      modal.classList.add("is-active");
+
+      modal.querySelector('.modal-background').addEventListener('click', function(e) {
+        e.preventDefault();
+        modal.classList.remove('is-active');
+      });
+      modal.querySelector('.delete').addEventListener('click', function(e) {
```

Comment:
more specific selector to be self documenting
`modal-card-head > button.delete`